### ISSUE TITLE
python eval.py get pool results ?

### ISSUE THREAD

**Issue by ccnankai** [argman/EAST#96]
hi @argman 
I use icdar2017 data to train, based on your results on icdar2015 and 2013（model.ckpt-49491）, but why the test results are zero?   0 text boxes before nms ?  This problem has troubled me for a long time, Can you help me solve this problem?   hankyou very much .

![image](https://user-images.githubusercontent.com/33537489/35190328-a2079a58-fe99-11e7-88b8-3647c7e50281.png)

**Comment by ccnankai** [comment]
@zxytim @QaisarRajput

**Comment by QaisarRajput** [comment]
This can be anything provide more info i.e sample of input data. but the first thing i can suggest is to look at the tensorboard of the trained model. more precisely the Images. (score_maps, geo_maps etc) those will get you an idea whether the model you trained is fine or not. 0 text boxes are weird..

**Comment by argman** [comment]
yes, I write all the training details in tensorboard

**Comment by ZezhouLi** [comment]
@argman @QaisarRajput I also tried to fine-tune the 49491 model on icdar 2017 chinese dataset, which has about 8000 training images, but after nearly 5000 steps of training, the total loss does not converge and dumping around 0.015. Then I tried to fine-tune from resnet model with the same dataset. Finally got the same dumping thing. Tensorboard details are listed below.
![image](https://user-images.githubusercontent.com/16214660/35614653-cbe067a6-06aa-11e8-9bb2-44aa5824688b.png)

![image](https://user-images.githubusercontent.com/16214660/35614723-fc868548-06aa-11e8-9bb8-82e927f65318.png)

I was wondering what went wrong here since the training does not converge and test results are poor?
